macros/Compat.ejs

```diff
@@ -698,7 +698,7 @@ if (!compatData) {
 traverseFeatures(compatData, depth, '');
 
 if (features.length > 0) {
-    output = '<div class="bc-data">';
+    output = `<div class="bc-data" data-id="{query}">`;
```

Comment:
As said in slack, `data-*` attributes are disallowed altogether in Kuma apparently. So, try to make this just `id`. Also missing a `$` in front of the template literal.
```suggestion
    output = `<div class="bc-data" id="${query}">`;
```

Reply:
How about `id="bcd_id__${query}"` instead? 
Since regular IDs have a "DOM meaning"

Reply:
I made it `id="bcd:${query}"` instead.